### ISSUE TITLE
FullTextExtrationService has NPE.

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/hydra-works#225]
```
#<NoMethodError: undefined method `size' for nil:NilClass>
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/hydra-works-7c9a10d339fa/lib/hydra/works/services/generic_file/full_text_extraction_service.rb:32:in `rescue in extract'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/hydra-works-7c9a10d339fa/lib/hydra/works/services/generic_file/full_text_extraction_service.rb:28:in `extract'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/hydra-works-7c9a10d339fa/lib/hydra/works/services/generic_file/full_text_extraction_service.rb:7:in `run'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-d5464453225d/curation_concerns-models/app/jobs/fulltext_extraction_job.rb:8:in `perform'
```

**Comment by jcoyne** [comment]
This happens when FullTextExtractionService is called without a FileSet and with a file path

**Comment by jcoyne** [comment]
Closed by #228
